### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewDebug_ScreenReader: Screenreader does not read the name of the "Breakpoints" toolbar as well as reads incorrect name for "Watch" toolbar.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52307]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Navigate to Activity Bar and select "View Debug"(Cntrl+Shift+D) button.
3. Navigate to toolbars section under heading "Debug".

**Actual:**
Screenreader does not read the name for the "Breakpoints" toolbar and reads the incorrect name for "Watch" toolbar as "Expressions" toolbar.

**Same issue has been observed with JAWS.**

**Expected:**
Screenreader should read name for the "Breakpoints" toolbar and read proper name for "Watch" toolbar not as "Expressions" toolbar.

**Recommendations:**
Provide proper names to the toolbars
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the correct names for all the toolbars present on the screen.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_ViewDebug_ScreenReader_ToolbarNames.pptx](https://github.com/Microsoft/vscode/files/2114747/A11y_VSCode_ViewDebug_ScreenReader_ToolbarNames.pptx)

Does this issue occur when all extensions are disabled?: Yes
